mcc/surgeon/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	_ "github.com/lib/pq"
```

Comment:
@miguelbernadi It's to avoid the gofmt error because this import is not used in this code directly, but this is required indirectly to make use of the Postgresql Go Driver.